coalib/output/ConsoleInteraction.py

```diff
@@ -255,7 +255,8 @@ def print_lines(console_printer,
             console_printer.print('')
         else:
             console_printer.print(highlight_text(
-                no_color, line[printed_chars:], lexer), end='')
+                no_color, line[printed_chars:], lexer, BackgroundMessageStyle),
+                                  end='')
```

Comment:
Hmm, I don't think we want that `style` variable from `highlight_text` be non-optional